GeneralsMD/Code/GameEngine/Source/GameLogic/AI/AIGroup.cpp

```diff
@@ -2795,23 +2795,21 @@ void AIGroup::groupCheer( CommandSourceType cmdSource )
 	*/
 void AIGroup::groupSell( CommandSourceType cmdSource )
 {
-	std::list<Object *>::iterator i, thisIterator;
-	Object *obj;
+	std::list<Object *>::iterator i;
+	std::vector<Object *> groupObjectsCopy;
+	groupObjectsCopy.reserve(m_memberListSize);
 
+	// TheSuperHackers @bugfix Mauller 26/06/2025 when sellObject is called, the objects in this AiGroup get removed from it. This happens within the Object::deselectObject() function.
+	// This invalidates the local AIGroup object from under the call to groupSell, therefore we need to make a local copy of the objects that need selling.
```

Comment:
Instead if "invalidates", you could write "deletes". Because that is what happens.